src/handlers/major_change.rs

```diff
@@ -167,13 +190,32 @@ pub(super) async fn handle_input(
 
             return Ok(());
         }
+        Invocation::ConcernsAdded => (
+            // Ideally, we would remove the `enabled_label` (if present) and add it back once all concerns are resolved.
+            //
+            // However, since this handler is stateless, we can't track when to re-add it, it's also a bit unclear if it
+            // should be re-added at all. Also historically the `enable_label` wasn't removed either, so we don't touch it.
+            format!("Concern(s) have been raised on the [associated GitHub issue]({}). This proposal is now blocked until those concerns are fully resolved.", event.issue.html_url),
+            None
+        ),
+        Invocation::ConcernsResolved => (
+            if event.issue.labels().contains(&Label {
+                name: config.accept_label.to_string(),
+            }) {
+                format!("All concerns on the [associated GitHub issue]({}) have been resolved, this proposal is no longer blocked, and will be approved in 10 days if no (new) objections are raised.", event.issue.html_url)
```

Comment:
Isn't `accept_label` only applied *after* the 10 day waiting period? :thinking:

Reply:
Yeah, should have been `enabling_label`. Fixed.